### DISCUSSION TITLE
关于在开始屏幕拖动图标黑屏

### DISCUSSION THREAD

**Discussion by YuI2333** [shuax/MouseInc.Settings#139] [Q&A]
系统：19044.1202
重现步骤为：打开 Mouselnc 右键随便划拉几下，然后在开始屏幕拖动图标调整位置的时候屏幕黑了1秒然后恢复正常。
虽然19044还没正式发布，不过应该已经转正，就等巨硬改个名发布了。
19043.1202并未测试，不想重装系统了，您嫌麻烦的话可以等正式版发布后有空了再做测试，目前并不影响使用。
最后感谢您为我们带来这么好的软件！

**Comment by shuax** [comment]
我已经在win11了，没有开始屏幕。
